release/release_tests.yaml

```diff
@@ -4903,6 +4921,21 @@
       cluster:
         cluster_compute: tpl_64_gce.yaml
 
+- name: microbenchmark_311
```

Comment:
instead of creating a new blob, can we use variations in the blob right above?

Reply:
good idea. let me try